app/models/vm/operations.rb

```diff
@@ -18,7 +18,11 @@ def cockpit_url
     miq_server = ext_management_system.nil? ? nil : ext_management_system.zone.remote_cockpit_ws_miq_server
     MiqCockpit::WS.url(miq_server,
                        MiqCockpitWsWorker.fetch_worker_settings_from_server(miq_server),
-                       ipaddresses.first)
+                       ipv4_address || ipaddresses.first)
+  end
+
+  def ipv4_address
+    ipaddresses.find { |ip| (IPAddr.new ip).ipv4? }
```

Comment:
Often times they say `IPAddr.new(ip).ipv4?`

However, it doesn't matter. Firstly, I hate to nitpick, and secondly, time to time it's good for developer to see something unusual, otherwise they stagnate. 👍